tests/functional/worker/metrics.test.js

```diff
@@ -106,62 +107,13 @@ function metricsValidObfuscationCreatesSM (type, browserVersionMatcher) {
     }
   )
 }
-
-function metricsInvalidObfuscationCreatesSM (type, browserVersionMatcher) {
```

Comment:
What's wrong with this test?

Reply:
https://issues.newrelic.com/browse/NEWRELIC-7142